src/main/java/com/tenx/ms/retail/stock/repository/StockRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.tenx.ms.retail.stock.repository;
+
+import com.tenx.ms.retail.stock.domain.StockEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+
+
+@Repository
+public interface StockRepository extends JpaRepository<StockEntity, Long>{
+    Optional<StockEntity> findOneByStore_StoreIdAndProductId(Long storeId, Long productId);
```

Comment:
It is not recommended to use _ in method name. for this case, you can change it to findOneByStoreIdAndProductId